### ISSUE TITLE
rumdl ignores inline config when reading from stdin

### ISSUE THREAD

**Issue by benjamingeer** [rvben/rumdl#445]
### rumdl version

0.1.23

### Installation method

cargo install

### Bug description

When rumdl reads the input file from stdin, it ignores inline configuration.

### Steps to reproduce

1. Create a Markdown file with this content:

```markdown
# A Test

<!-- rumdl-disable-next-line MD013 -->
This is a very long line with a lot of words and its purpose is to test linting behaviour when rumdl gets its input from stdin instead of by reading a file.
```

Run rumdl like this:

```sh
rumdl check - < test.md
```

Then run it like this:

```sh
rumdl check test.md
```

### Expected behavior

In both cases, the output should be `Success: No issues found in 1 file`.

### Actual behavior

In the first case, the output is:

```text
<stdin>:4:81: [MD013] Line length 156 exceeds 80 characters

Found 1 issue(s) in <stdin>
```

In the second case, the output is:

```text
Success: No issues found in 1 file (10 ms)
```

### Configuration

None.

### Operating system

Linux

### Additional context

[nvim-lint](https://github.com/mfussenegger/nvim-lint/blob/master/lua/lint/linters/rumdl.lua) invokes rumdl using stdin.

**Comment by rvben** [comment]
Thanks for reporting this, @benjamingeer! This is fixed in v0.1.24. Inline config directives (`rumdl-disable-next-line`, `rumdl-disable`/`rumdl-enable`, etc.) now work when linting from stdin, the same as they do for files.

**Comment by benjamingeer** [comment]
Many thanks for the quick fix and the great program!
